README.md

```diff
@@ -3,8 +3,6 @@
 <div align="center">
 
 [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg)](https://snyk.io/test/github/bbc/psammead)
-[![dependencies Status](https://david-dm.org/bbc/psammead/status.svg)](https://david-dm.org/bbc/psammead)
-[![devDependencies Status](https://david-dm.org/bbc/psammead/dev-status.svg)](https://david-dm.org/bbc/psammead?type=dev)
```

Comment:
Do we need to remove this?

Reply:
No - I'm proposing we remove them though, because they're broken and distract from an otherwise well-structured document. See: https://github.com/bbc/psammead.

Reply:
Yeah they look pretty broken, be good to replace them I guess?